bot/seasons/evergreen/error_handler.py

```diff
@@ -42,9 +42,12 @@ def __init__(self, bot):
                 f"{ctx.author} called the command '{ctx.command}' "
                 "but they were on cooldown!"
             )
+            seconds = error.retry_after
+            remaining_minutes, remaining_seconds = divmod(seconds, 60)
+            error_message = f'{int(remaining_minutes)} minutes {math.ceil(remaining_seconds)} seconds'

```

Comment:
This is not an error message. Why did you call it that? Variables should be named something that logically corresponds with what they are.